keyboards/melody96/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```

Comment:
This rule will inherit from `keyboards/melody96/rules.mk`, so it's redundant here.

```suggestion
```